.stylelintrc.yaml

```diff
@@ -0,0 +1,256 @@
+extends: stylelint-config-standard
+ignoreFiles:
+  - node_modules/**/*
+  - packages/**/node_modules/**/*
+plugins:
+  - stylelint-selector-bem-pattern
+  - stylelint-scss
+rules:
+  # Follow best practices
+  font-family-name-quotes: always-where-recommended
+  # http://stackoverflow.com/questions/2168855/is-quoting-the-value-of-url-really-necessary
+  function-url-quotes: never
+  # https://www.w3.org/TR/selectors/#attribute-selectors
+  # http://stackoverflow.com/q/3851091
+  selector-attribute-quotes: always
+  # Single-quotes are a convention throughout our codebase
+  string-quotes: single
```

Comment:
We've been doing double quotes in CSS and single quotes in JS, but I suppose single quotes everywhere works too.